homeassistant/__main__.py

```diff
@@ -262,6 +263,8 @@ def main() -> int:
     """Start Home Assistant."""
     validate_python()
 
+    faulthandler.enable()
```

Comment:
this prints to stderr and nobody will see that. I also wonder if it helps to always enable, or, since it can be enabled with an environment variable, it's better that people use it when a segmentation fault occurs.

Reply:
@balloob This is a good point. But even under an hypervisor this will be very hard to see.
The function faulthandler.enable() can take a file descriptor. Ideally we would like to send the file descriptor for `home-asistant.log` instead of the default `stderr`.
```
faulthandler.enable(file_descriptor_for_homeassistant_log)
```
Do you know where in the code is this descriptor created?

Reply:
core/homeassistant/bootstrap.py

Reply:
> core/homeassistant/bootstrap.py

Thanks, I have added code there.